src/ilsresc.cpp

```diff
@@ -490,6 +497,8 @@ main( int argc, char **argv ) {
     rcComm_t *Conn = rcConnect( myEnv.rodsHost, myEnv.rodsPort, myEnv.rodsUserName,
                       myEnv.rodsZone, 0, &errMsg );
 
+    const auto disconnect = irods::at_scope_exit{[&Conn] { rcDisconnect(Conn); }};
```

Comment:
If `Conn` is `NULL`, we do not need to call `rcDisconnect`. Why move it here?

Reply:
Got it. Thanks

Reply:
I'm going to leave this as is since it's what was tested.

We can always adjust later.